src/features/session_trace/aggregate/trace/storage.js

```diff
@@ -265,38 +200,44 @@ export class TraceStorage {
     this.#storeSTN(new TraceNode('Ajax', metrics.time, metrics.time + metrics.duration, `${params.status} ${params.method}: ${params.host}${params.pathname}`, 'ajax'))
   }
 
-  /* Below are the interface expected & required of whatever storage is used across all features on an individual basis. This allows a common `.events` property on Trace shared with AggregateBase.
-    Note that the usage must be in sync with the EventStoreManager class such that AggregateBase.makeHarvestPayload can run the same regardless of which storage class a feature is using. */
-  isEmpty () {
-    return this.nodeCount === 0
-  }
-
-  save () {
-    this.#backupTrace = this.trace
+  trimSTNsByTime (lookbackDuration = ERROR_MODE_SECONDS_WINDOW) {
+    this.parent.events.clear({
+      clearBeforeTime: Math.max(now - lookbackDuration, 0),
+      timestampKey: 'e'
+    })
   }
 
-  get () {
```

Comment:
What happened to get? I'm not sure making ST so specialized like SR when it had a common interface to all the other features is the right move here. This pattern deviation is more devl bug prone. It also seems much beyond the scope of the PR ticket.

Reply:
Because Storage.js is no longer a state retrieval device.  it simply is a formatter for storing in the native `this.events` now, which means there's no point for things like get. I think we need to have a meeting on this to ensure that the changes make sense as to what they are doing.